src/file.cr

```diff
@@ -439,7 +439,7 @@ class File < IO::FileDescriptor
   # File.read("bar") # => "foo"
   # ```
   def self.read(filename, encoding = nil, invalid = nil) : String
-    File.open(filename, "r") do |file|
+    open(filename, "r") do |file|
```

Comment:
For this case and the other below, this is not an issue I think to use `File.` because the returned object is not a `File`.

However it's true that if someone wants to override the behavior of `open`, it is needed to call the implicit `open` in other class methods.. So maybe it's good, I don't know

Reply:
The explicit receiver is unnecessary and can be safely omitted.